README.md

```diff
@@ -2,46 +2,108 @@
 
 EPFL ENAC-IT Continuous Deployment
 
+This action makes a POST request to a given URL to deploy the app and then do a loop
+to another URL until the status is `finished` or `error`
+Initially created for checking if a enac deployment is complete (via enacit-ansible)
+
+
+## Create two secrets in your repository
+
+- under your repository settings in /settings/secrets/actions
+- Add DEPLOYMENT_SECRET and DEPLOYMENT_ID given by the linux enacit-linux-sysadmins@epfl.ch
+
 ## Add Continuous Deployment to PROD hosting
 
 Create `.github/workflows/deploy-prod.yml` containing :
 
 ```yml
-# https://github.com/EPFL-ENAC/github-actions-runner#readme
+# https://github.com/EPFL-ENAC/epfl-enac-deploy-action#readme
 name: deploy-prod
 
-on:
+"on":
   push:
     branches:
       - main
 
 jobs:
   deploy:
-    runs-on: [self-hosted, deploy, prod]
+    runs-on: ubuntu-latest
     steps:
       - uses: EPFL-ENAC/epfl-enac-deploy-action@main
         with:
-          token: ${{ secrets.GITHUB_TOKEN }}
+          ENAC_IT4R_CD_environment: "prod"
+          ENAC_IT4R_CD_deployment_secret: ${{ secrets.DEPLOYMENT_SECRET }}
+          ENAC_IT4R_CD_deployment_id: ${{ secrets.DEPLOYMENT_ID }}
 ```
 
 ## Add Continuous Deployment to TEST hosting
 
 Create `.github/workflows/deploy-test.yml` containing :
 
 ```yml
-# https://github.com/EPFL-ENAC/github-actions-runner#readme
+# https://github.com/EPFL-ENAC/epfl-enac-deploy-action#readme
 name: deploy-test
 
-on:
+"on":
   push:
     branches:
-      - develop
+      - dev
 
 jobs:
   deploy:
-    runs-on: [self-hosted, deploy, test]
+    runs-on: ubuntu-latest
     steps:
       - uses: EPFL-ENAC/epfl-enac-deploy-action@main
         with:
-          token: ${{ secrets.GITHUB_TOKEN }}
+          ENAC_IT4R_CD_environment: "dev"
+          ENAC_IT4R_CD_deployment_secret: ${{ secrets.DEPLOYMENT_SECRET }}
+          ENAC_IT4R_CD_deployment_id: ${{ secrets.DEPLOYMENT_ID }}
+```
+
+## Inputs
+
+### `ENAC_IT4R_CD_environment`
+
+The environement to deploy to: dev or prod
+
+### `ENAC_IT4R_CD_deployment_secret`
+  - The secret associated with the deployment_id
+  - only viable for that unique id
+
+### `ENAC_IT4R_CD_deployment_id`
+
+The unique id associated with the deployment:
+  - maybe the fqdn where it will be deployed
+  - maybe a unique id (uuid4 or anything)
```

Comment:
It's value will be chosen and given by ENAC-IT's sysadmins at the same time as the `ENAC_IT4R_CD_deployment_secret`.
Yes, it'll be the fqdn, but I'd simply say:

The unique deployment id received by ENAC-IT's sysadmins.